### ISSUE TITLE
move readme documentation 'further reading' links to a 'further reading' subheading section

### ISSUE THREAD

**Issue by brandondees** [devpunks/snuggsi#2]
README is progressing but is turning into a big flat wall of text pretty quickly, hard to parse quickly by scanning through to find what you want. establishing some conventional structures that can be repeated across subsections can make those sections easier to grok while skimming through. specifically I'm noticing that almost every topical section has a series of links to things like specification documents and tutorials, usually near the end. why not group those into a tidy little list under a "Further Reading" subheading for each topic, so that it's easy to filter that out if you're aiming to focus on the TL;DR bits, or easy to focus on if you're really looking to dig into a topic.

**Comment by snuggs** [comment]
We now have a better pruned README.md
https://github.com/devpunks/snuggsi

Also "Further Reding / TL;DR;" Concepts moved to wiki.
https://github.com/devpunks/snuggsi/wiki
